mvnw

```diff
@@ -281,8 +281,7 @@ if [ -n "$wrapperSha256Sum" ]; then
 fi
 
 #This option is required by the javalite db-migrator-maven-plugin, which does not support adding VM options in its config
-REQUIRED_BUILD_OPTIONS='--add-opens=java.base/java.net=ALL-UNNAMED'
-MAVEN_OPTS="$(concat_lines "$MAVEN_PROJECTBASEDIR/.mvn/jvm.config") $REQUIRED_BUILD_OPTIONS $MAVEN_OPTS"
+MAVEN_OPTS="$(concat_lines "$MAVEN_PROJECTBASEDIR/.mvn/jvm.config") $MAVEN_OPTS"
```

Comment:
do NOT modify the generated mvnw wrapper script .. this will be wiped whenever we upgrade the wrapper

Reply:
this changes it back to the generated version

Reply:
except for the comment - let me regenerate the wrapper

Reply:
oh .. good